src/Ng/Jobs/Catalog2LuceneJob.cs

```diff
@@ -107,6 +107,13 @@ protected override void Init(IDictionary<string, string> arguments, Cancellation
             }
         }
 
+        ReadCursor GetTheLeastAdvancedRegistrationCursor(string registrationArg, CancellationToken cancellationToken)
+        {
+            string[] registrations = registrationArg.Split(';');
```

Comment:
There will be one registration in index.json ; its location will be masked by a dns. The job will need to read the cursor blob location.